components/brave_rewards/resources/rewards_panel/stories/index.tsx

```diff
@@ -26,13 +25,13 @@ export default {
 
 const locale = createLocaleContextForTesting(localeStrings)
 
-function actionLogger (name: string) {
+function actionLogger(name: string) {
```

Comment:
Curious: do we no longer use the space between function name and args in our JS styles?

Reply:
tbh I'm not really sure - this is just what the default formatter in vscode gave me. That said, I think our style is meant to be the `prettier` formatting based on `.prettierrc.js` but its not enabled by default for most directories so I'm not sure what I should do here - could revert this, I guess?

Reply:
To be honest, I would love it if we could move away from that convention - the function definition style should match the call style: `function f() {}` / `f()`. But I don't think it's my call. 😄

Reply:
I think `prettier` doesn't use it so we **are** moving away from it but not very quickly :laughing: 

cc @petemill for a call on this (I'm keen to move away from it too)

Reply:
Yes I think it's ok to keep that rule and make that decision - we should probably get on to formatting our files with those rules! I haven't heard anyone on the team argue for the space-style recently.